cypress/e2e/specialFeatures/liteSiteWeight/assertions/liteSiteWeight.js

```diff
@@ -0,0 +1,48 @@
+import interceptGetRequests from '../helpers/interceptGetRequests';
+
+const MAX_PAGE_WEIGHT = 100;
+
+export default ({ path, pageType }) => {
+  describe('Page weight', () => {
+    const allRequests = [];
+    const getPageSizeInBytes = request => {
+      const getSize = `echo ${JSON.stringify(request)} | gzip | wc -c`;
+      if (typeof request === 'string') {
+        return cy.exec(getSize).then(result => {
+          return Number.isNaN(result.stdout)
+            ? 0
+            : parseFloat(result.stdout) / 1024;
+        });
+      }
+      return cy.wrap(0);
+    };
```

Comment:
This feels like more of a helper - I wonder if we should consider moving it into the helpers file?